src/Accounts/Accounts/CommonModule/AzModule.cs

```diff
@@ -133,19 +134,40 @@ public async Task EventListener(string id, CancellationToken cancellationToken,
                     await OnProcessRecordAsyncEnd(id, cancellationToken, getEventData, signal, processRecordId);
```

Comment:
Shall `processRecordId` be changed?

Reply:
No need to change since in function OnProcessRecordAsyncEnd, the processRecordId is actually a redundant parameter without being used.